src/main/webapp/WEB-INF/tags/instructor/results/filterEditModal.tag

```diff
@@ -0,0 +1,120 @@
+                            ${fn:escapeXml(section)}
+                        </option>
+                      </c:forEach>
+                      <option value="None"<c:if test="${filterPanel.noneSectionSelected}"> selected</c:if>>
+                        <%=Const.NO_SPECIFIC_RECIPIENT%>
+                      </option>
+                    </select>
+                  </div>
+                </div>
+              </c:if>
+            </div>
+            <div class="col-md-5">
+              <label class="control-label">
+                Additional settings:
+              </label>
+              <div data-toggle="tooltip" title="Group results in the current view by team">
+                <div class="checkbox">
+                  <label<c:if test="${filterPanel.sortType == 'question'}"> class="text-strike"</c:if>>
+                    <input type="checkbox" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYTEAM%>"
+                           id="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYTEAM%>"
+                           <c:if test="${filterPanel.groupedByTeam}">checked</c:if>
+                           <c:if test="${filterPanel.sortType != 'question'}">class="checkbox-group-by-team"</c:if>> Group by Teams
+                  </label>
+                </div>
+              </div>
+              <div data-toggle="tooltip" title="Show statistics">
+                <div class="checkbox">
+                  <label<c:if test="${filterPanel.sortType == 'recipient-giver-question' or filterPanel.sortType == 'giver-recipient-question'}"> class="text-strike"</c:if>>
+                    <input type="checkbox" id="show-stats-checkbox"
+                           name="<%=Const.ParamsNames.FEEDBACK_RESULTS_SHOWSTATS%>"
+                    <c:if test="${filterPanel.statsShown}"> checked</c:if>> Show Statistics
+                  </label>
+                </div>
+              </div>
+              <div data-toggle="tooltip" title="Indicate missing responses">
+                <div class="checkbox">
+                  <input type="checkbox"
+                         id="indicate-missing-responses-checkbox"
+                         value="true"
+                         name="<%=Const.ParamsNames.FEEDBACK_RESULTS_INDICATE_MISSING_RESPONSES%>"
+                  <c:if test="${filterPanel.missingResponsesShown}"> checked</c:if>>
+                  Indicate Missing Responses
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_SESSION_NAME%>"
+                 value="${filterPanel.feedbackSessionName}">
+          <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>"
+                 value="${filterPanel.courseId}">
+          <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>"
+                 value="${data.account.googleId}">
+          <hr>
+          <div class="row">
```

Comment:
Here we also need to manage the size of the button to be consistent with buttons above. With your approach it's will be hard to do so. Also, here used grid system from bootstrap, so the button will be responsive and look good regarding screen resolution.